package.json

```diff
@@ -26,10 +26,11 @@
     "postinstall": "lerna bootstrap && npm run -s update-greenkeeper-json",
     "prerelease": "npm run build:full && npm run mocha && npm run lint",
     "release": "lerna version && lerna publish from-git --yes",
+    "update-package-lock-json": "lerna exec \"npm install -s --ignore-scripts --package-lock-only --no-audit\"",
     "update-template-deps": "node bin/update-template-deps -f",
     "update-greenkeeper-json": "node bin/update-greenkeeper-json -f",
     "sync-dev-deps": "node bin/sync-dev-deps",
-    "version": "npm run update-template-deps && npm run apidocs",
+    "version": "npm run update-package-lock-json && npm run update-template-deps && npm run apidocs",
```

Comment:
Does running `update-template-deps` before or after `update-package-lock-json` make any difference to the `package-lock.json` files that are updated?

Reply:
Not really.